### PR TITLE
Review pages - px to rem conversion

### PR DESCRIPTION
**Pull request by rnou112**
New component for common layout with side image
Completed px to rem conversion

### CODE REVIEW

File: assets/_layout.scss [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 .subpage {
   background: #fff;
-  border: 1px solid #dcdfe6;
-  margin: 1em 0;
-  padding: 1em;
-  @media (min-width: 48em) {
-    margin: 2.5em 0;
-   padding: 2em;
-}
+  border: 0.06rem  solid #dcdfe6;
```
Comment (alan-wu) [inline assets/_layout.scss:17]: Same here, set it to 1px.

File: assets/_common.scss [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+  padding: 2.5rem 2rem;
 }
 
 .curved-box{
   background: #FFFFFF 0% 0% no-repeat padding-box;
-  box-shadow: 0px 2px 5px #0000001A;
-  border: 1px solid $lineColor1;
-  border-radius: 20px;
+  box-shadow: 0rem 0.13rem 0.31rem #0000001A;
+  border: 0.06rem solid $lineColor1;
```
Comment (alan-wu) [inline assets/_common.scss:21]: Probably keep this as 1px. We do not want the border get round down to zero.

File: components/Events/LatestEvents.vue [hunk 3]
```diff
@@ -132,44 +133,38 @@ export default {
...
-            grid-gap: 0px;
+            grid-gap: 0rem;
             .grid-item {
-                padding: 12px;                            
-                border-top: 1px solid #dfdfdf;
-                border-left: 1px solid #dfdfdf;
-                font-size:16px;
-                line-height:18px;
+                padding: 0.75rem;                            
+                border-top: 0.06rem solid #DFDFDF;
```
Comment (alan-wu) [inline components/Events/LatestEvents.vue:164]: Same here.

File: components/Events/LatestEvents.vue [hunk 4]
```diff
@@ -132,44 +133,38 @@ export default {
...
+                border-left: 0.06rem solid #DFDFDF;
```
Comment (alan-wu) [inline components/Events/LatestEvents.vue:165]: Here as well.

File: components/SocialBox.vue [hunk 5]
```diff
@@ -37,17 +37,17 @@ export default {
...
-    height:28px;
-    width:28px;
-    padding:6px;    //TBC
+    height:1.75rem;
+    width:1.75rem;
+    padding:0.38rem;    
     
     border-radius:50%;
-    border:1.5px solid $app-primary-color;
+    border:0.1rem solid $app-primary-color;
```
Comment (alan-wu) [inline components/SocialBox.vue:50]: It is fine to set this to 1px.

File: pages/news-and-events/events/_detail.vue [hunk 6]
```diff
@@ -80,48 +80,43 @@ export default {
...
 
     .event-detail{
-        padding-top:32px;
-        padding-bottom:25px;    
-        border-top: 1px solid $lineColor1;
-        border-bottom: 1px solid $lineColor1;
-        margin-bottom:20px;     
+        padding-top:2rem;
+        padding-bottom:1.56rem;    
+        border-top: 0.06rem solid $lineColor1;
```
Comment (alan-wu) [inline pages/news-and-events/events/_detail.vue:110]: 1px please.

File: pages/news-and-events/events/_detail.vue [hunk 7]
```diff
@@ -80,48 +80,43 @@ export default {
...
+        border-bottom: 0.06rem solid $lineColor1;
```
Comment (alan-wu) [inline pages/news-and-events/events/_detail.vue:111]: Same here.

File: pages/news-and-events/news/_detail.vue [hunk 8]
```diff
@@ -68,46 +68,43 @@ export default {
...
 
     .news-detail{
-        padding-top:32px;
-        padding-bottom:25px;        
-        border-top: 1px solid $lineColor1;
-        border-bottom: 1px solid $lineColor1;
-        margin-bottom:20px;     
+        padding-top:2rem;
+        padding-bottom:1.56rem;        
+        border-top:0.06rem solid $lineColor1;
```
Comment (alan-wu) [inline pages/news-and-events/news/_detail.vue:98]: Here.

File: pages/news-and-events/news/_detail.vue [hunk 9]
```diff
@@ -68,46 +68,43 @@ export default {
...
+        border-bottom: 0.06rem solid $lineColor1;
```
Comment (alan-wu) [inline pages/news-and-events/news/_detail.vue:99]: Here.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good in general but some borders should be kept at 1px.

File: assets/_layout.scss (hunk 1 above)
Comment (rnou112) [reply assets/_layout.scss:17]: You mean to set border back to 1px instead of 0.06rem?

**Comment by rnou112** [comment]
Converted borders back to pixels

**Review by alan-wu — APPROVED** [review]
Thanks, looks great.

**Pull request merged**